setup.py

```diff
@@ -17,7 +17,36 @@
 
 from setuptools import find_packages, setup
 
-install_requires = [
+DESCRIPTION = (
+    "Blue Brain text mining toolbox for semantic search and information extraction"
+)
+
+LONG_DESCRIPTION = """
+Blue Brain Search is a text mining toolbox to perform semantic literature search 
+and structured information extraction from text sources.
+
+This project originated from the Blue Brain Project efforts on exploring and 
+mining the CORD-19 dataset."""
+
+CLASSIFIERS = [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)",
+    "Operating System :: Unix",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Scientific/Engineering :: Image Processing",
```

Comment:
I don't think this topic is relevant for BBS, isn't it?

Reply:
Nice catch, thank you very much! Fixed in ff77a07